tests/migrations/test_autodetector.py

```diff
@@ -741,6 +741,17 @@ class AutodetectorTests(BaseAutodetectorTests):
             ("publishers", models.ManyToManyField("testapp.Publisher", blank=True)),
         ],
     )
+    magazine = ModelState(
+        "testapp", "Magazine", [("id", models.AutoField(primary_key=True))]
+    )
+    author_with_m2m_magazine = ModelState(
+        "testapp",
+        "Author",
+        [
+            ("id", models.AutoField(primary_key=True)),
+            ("publishers", models.ManyToManyField("testapp.Magazine")),
+        ],
+    )
```

Comment:
I think we only need a variable for something we need to use more than once.
```suggestion
    other_publisher = ModelState(
        "testapp",
        "OtherPublisher",
        [
            ("id", models.AutoField(primary_key=True)),
        ],
    )
```